cfg_dataset/cfg.py

```diff
@@ -1,68 +1,75 @@
-import os
-from collections import defaultdict
-from random import choice
+import sys
+import nltk
```

Comment:
[just a reminder] nltk is external lib, so that import should be in the group below [https://peps.python.org/pep-0008/#imports](https://peps.python.org/pep-0008/#imports)